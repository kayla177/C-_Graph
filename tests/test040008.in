ENTITY BW0DPLY1 name_BW0D journal
ENTITY 2X7DMY1U name_2X7D paper
ENTITY 90OMIQYT name_90OM conference
ENTITY PQBWSZ1Y name_PQBW paper
ENTITY ERODZ6JX name_EROD conference
ENTITY YEKR2TFK name_YEKR paper
ENTITY U634DJRD name_U634 paper
ENTITY AYGJWT35 name_AYGJ conference
ENTITY 038CQ66P name_038C journal
ENTITY HM6XW440 name_HM6X institution
RELATIONSHIP 90OMIQYT presented_at HM6XW440 1.4
RELATIONSHIP YEKR2TFK published_in BW0DPLY1 2.0
RELATIONSHIP 038CQ66P authored_by YEKR2TFK 3.1
RELATIONSHIP 038CQ66P cited_by HM6XW440 3.0
RELATIONSHIP U634DJRD presented_at HM6XW440 4.3
RELATIONSHIP AYGJWT35 published_in 038CQ66P 7.4
RELATIONSHIP PQBWSZ1Y collaborated_with U634DJRD 1.7
RELATIONSHIP 038CQ66P authored_by BW0DPLY1 2.7
RELATIONSHIP 90OMIQYT affiliated_with YEKR2TFK 9.9
RELATIONSHIP ERODZ6JX affiliated_with 2X7DMY1U 3.0
RELATIONSHIP 90OMIQYT authored_by 2X7DMY1U 9.0
RELATIONSHIP AYGJWT35 affiliated_with U634DJRD 3.6
RELATIONSHIP PQBWSZ1Y cited_by YEKR2TFK 3.7
RELATIONSHIP ERODZ6JX affiliated_with HM6XW440 4.8
RELATIONSHIP 90OMIQYT authored_by U634DJRD 2.7
HIGHEST
PRINT 90OMIQYT
PRINT ERODZ6JX
PRINT YEKR2TFK
HIGHEST
HIGHEST
HIGHEST
HIGHEST
DELETE HM6XW440
DELETE 038CQ66P
EXIT
