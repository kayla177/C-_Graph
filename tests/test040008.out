success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
2X7DMY1U AYGJWT35 29.4
HM6XW440 YEKR2TFK 2X7DMY1U U634DJRD
2X7DMY1U HM6XW440
BW0DPLY1 038CQ66P 90OMIQYT PQBWSZ1Y
2X7DMY1U AYGJWT35 29.4
2X7DMY1U AYGJWT35 29.4
2X7DMY1U AYGJWT35 29.4
2X7DMY1U AYGJWT35 29.4
success
success
